articles/sentinel/customer-managed-keys.md

```diff
@@ -115,9 +95,9 @@ PUT https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{
 } 
 ```
 
-### Step 6: Contact the Microsoft Sentinel Product group to confirm onboarding 
+### Step 4: Contact the Microsoft Sentinel Product group to confirm onboarding 
 
-Lastly, you must confirm the onboarding status of your CMK enabled workspace by contacting the [Microsoft Sentinel Product Group](mailto:onboardrecoeng@microsoft.com).
+Lastly, confirm the onboarding status of your CMK enabled workspace by contacting the [Microsoft Sentinel Product Group](mailto:onboardrecoeng@microsoft.com).
```

Comment:
Would it be worth it to ask them to check the GET request first to confirm onboarding is completed?